pyigm/surveys/dlasurvey.py

```diff
@@ -30,6 +36,84 @@ class DLASurvey(IGMSurvey):
 
     """
     @classmethod
+    def load_HST16(cls, sample='stat'):
+        """ HST Survey by Neeleman+16
+
+        Parameters
+        ----------
+        sample : str, optional
+
+        Returns
+        -------
+        dla_survey
+
+        """
+        # Read DLAs
+        dlas = Table.read(pyigm_path + '/data/DLA/HST/HSTDLA.dat', format='ascii')
+
+        # Read Quasars
+        #qsos = Table.read(pyigm_path + '/all_qso_table.txt', format='ascii')
+
+        # Read Sightlines
+        survey = Table.read(pyigm_path + '/data/DLA/HST/hstpath.dat', format='ascii')
+
+        # Add info to DLA table
+        ras, decs, zems = [], [], []
+        for dla in dlas:
+            mt = np.where(survey['QSO'] == dla['NAME'])[0]
+            if len(mt) == 0:
+                pdb.set_trace()
+                raise ValueError("Uh oh")
+            else:
+                mt = mt[0]
+            # Generate RA/DEC
+            row = survey[mt]
+            coord = ltu.radec_to_coord('J{:02d}{:02d}{:f}{:s}{:02d}{:02d}{:f}'.format(
+                row['RAh'], row['RAm'], row['RAs'], row['DE-'], row['DEd'], row['DEm'],
+                row['DEs']))
+            ras.append(coord.ra.value)
+            decs.append(coord.dec.value)
+            # zem
+            zems.append(row['ZEM'])
+        dlas['RA'] = ras
+        dlas['DEC'] = decs
+        dlas['QSO_ZEM'] = zems
+
+        # Instantiate
+        dla_survey = cls.from_sfits(dlas)
+        dla_survey.ref = 'Neeleman+16'
+
+        # Fiddle a bit
+        survey.rename_column('STTMIN', 'Z_START')
+        survey.rename_column('STTMAX', 'Z_END')
+        stat = survey['Z_END'] > 0
+        stat_survey = survey[stat]
```

Comment:
need this line of code as otherwise statistical flag equal to 2 is also counted:
if sample == 'stat':
            stat_survey = stat_survey[stat_survey['F_STT'] == 1]